packages/tpetra/core/src/Tpetra_CrsMatrix_def.hpp

```diff
@@ -5927,42 +5927,36 @@ namespace Tpetra {
         Distributor& dist) const
   {
```

Comment:
That's fine :-) .  We can always defer merging the pull request until we have the desired interface.